housekeeping-vacuum-tool/README.md

```diff
@@ -65,6 +65,8 @@ The `dry-run` option allows you to observe the status of paths on the file syste
 |`housekeeping.datasource.username` |No| Username for your database.|
 |`housekeeping.datasource.password` |No| Password for your database.|
 |`housekeeping.db-init-script`|No|A file containing a script to initialise your schema can be provided if it does not already exist. Defaults to `classpath:/schema.sql`.|
+|`tables-validation`|No| Configuration for checking Hive table properties, prevent accidentally running the vacuum tool on a table that should not be vaccuumed. We always recommend running with `--dry-run=true` first and carefully look at the results.|
```

Comment:
Controls whether to check for the presence of Hive table properties to determine whether to run the vacuum tool or not. The properties can be configured by using the `tables-validation.hive-table-properties` configuration property (see below). This can be used to prevent 
accidentally running the vacuum tool on a table that should not be vacuumed. We always recommend running with `--dry-run=true` first and carefully looking at the results. Default is `true`.

Reply:
ok rewording aside
I guess it is not clear. There is no default option this is not something you can set like `tables-validation: true`. It's just the start of the section, I'll remove it and move the explanation to the `tables-validation.ive-table-properties`. I had another propery you could set before it made more sense then